# central_management_params.yaml
trajectory_planner_action_server:
  ros__parameters:
    map_data:
      map_path: "/ros2_ws/src/laneletMaps/HS-SchmalkaldenPart1.osm"
